Application/mobile/app_code/lib/l10n/app_fr.arb

```diff
@@ -112,6 +116,7 @@
   "myInformation": "Mes informations",
   "myRents": "Mes locations",
   "myTickets": "Mes tickets",
+  "new_": "Nouveau",
```

Comment:
encore ici pour _ avant new

Reply:
Because new is a key word for Flutter, so I put a '_', otherwise I couldn't use it